tests/functional/includes/date.mock.js

```diff
@@ -0,0 +1,14 @@
+// Highjack the Date object so that our tests don't suddenly start failing...
```

Comment:
We could hardcode the date for the example, and that might be better than using `Date.now()` that we use today.